packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -8,7 +8,7 @@ import { nodeResolve } from "@rollup/plugin-node-resolve";
 import replace from "rollup-plugin-re";
 import typescript from "@rollup/plugin-typescript";
 import url from "@rollup/plugin-url";
-import { red, yellow } from "colors";
+import { red, yellow,blue } from "colors";
```

Comment:
It’s missing `npm run prettier`

Reply:
running prettier project wide causes 28 file changes. Is that fine?

Reply:
Nope. Then just add a space after `,`